public/editorialsupportteams.js

```diff
@@ -0,0 +1,41 @@
+import 'whatwg-fetch';
+
+function addNewStaff(team) {
+    var name = document.getElementById("name-entry-"+team).value;
+    var endpoint = "/api/editorialSupportTeams?name="+name+"&team="+team;
```

Comment:
As this is passed through webpack, this can be simplified further to `` `/api.editorialSupportTeams?name=${name}&team=${team}\` `` Check out [template literals](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Template_literals)